package.json

```diff
@@ -15,7 +16,7 @@
   },
   "pnpm": {
     "overrides": {
-      "hardhat@2>@nomicfoundation/edr": "workspace:*"
+      "hardhat>@nomicfoundation/edr": "workspace:*"
```

Comment:
Remove `@2` to make sure it matches pre-release versions as well. I tested it locally that removing the `@2` matches pre-release versions as well.

Reply:
Ah cool, I don't know why I didn't do it this way in the first place.